Geta.404Handler.sln

```diff
@@ -1,11 +1,20 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27130.2027
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29911.84
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "BVNetwork.404Handler", "BVNetwork.404Handler.csproj", "{DFBF0553-7E44-44BF-A59F-F94F9493FC17}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Geta.404Handler", "src\Geta.404Handler\Geta.404Handler.csproj", "{DFBF0553-7E44-44BF-A59F-F94F9493FC17}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "BVNetwork.404Handler.Tests", "..\tests\BVNetwork.404Handler.Tests\BVNetwork.404Handler.Tests.csproj", "{1E8472FF-8DBC-4BDF-BB96-F68A0598C7CF}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Geta.404Handler.Tests", "tests\Geta.404Handler.Tests\Geta.404Handler.Tests.csproj", "{1E8472FF-8DBC-4BDF-BB96-F68A0598C7CF}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{85A82003-EBCE-48FF-B34F-52AEFAF209A8}"
+	ProjectSection(SolutionItems) = preProject
+		CHANGELOG.md = CHANGELOG.md
+		LICENSE = LICENSE
+		pack.ps1 = pack.ps1
+		pack_cms10.ps1 = pack_cms10.ps1
```

Comment:
I see that CMS 10 support is dropped (it is ok). So remove also the script for packaging it.